### DISCUSSION TITLE
issue

### DISCUSSION THREAD

**Discussion by Tolulope25** [community/community#106339] [New to GitHub]
### Select Topic Area

General

### Feature Area

Issues

### Body

i opened an account and did all neccessary configurations, i can my repository but can't find the files in it

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by gil-son** [comment]
Hello,

Give more details to support you

After you created your repository you was able to connect your computer with the project on Github?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Tolulope25 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by theburcu** [comment]
Well I think you did create the repository but was not able to push the files and folders from your local machine to the GitHub Repository. Did you follow the steps correctly? Did you push to the origin?
